multi_tracker/Trackedobject object 
LedRegionData[] led_region_data
uint64 index
